docs/publish/index.md

```diff
@@ -0,0 +1,67 @@
+# ARCHER2 and publications
+
+This section provides information on how to acknowledge the use of ARCHER2 in your published work and
+how to register your work on ARCHER2 into the ARCHER2 publications database via SAFE.
+
+## Acknowledging ARCHER2
+
+We will shortly be publishing a description of the ARCHER2 service with a DOI that you can cite
+in your published work that arises from the use of ARCHER2. Until that time, please add the following
+words to any work you publish that arises from your use of ARCHER2:
+
+> This work used the ARCHER2 UK National Supercomputing Service (https://www.archer2.ac.uk).
+
+You should also tag outputs with the keyword "ARCHER2" whenever possible.
+
+## ARCHER2 publication database
+
+The ARCHER2 service maintains a publication database of works that have arisen from ARCHER2 and
+links them to project IDs that have ARCHER2 access. We ask all users of ARCHER2 to register any
+publications in the database - all you need to do this is your publication's DOI.
```

Comment:
I find the sentence _all you need to do this is your publication's DOI_ difficult to read. I'd use:
- _all you need is your publication's DOI._
- _you only need your publication's DOI._
The rest looks good to me.

Reply:
Thanks for the suggestion. Text has been changed.